tests/collecting/ImportersTest.py

```diff
@@ -2,7 +2,35 @@
 import unittest
 from collections import OrderedDict
 
-from coalib.collecting.Importers import import_objects
+from coalib.collecting.Importers import _import_module, import_objects
+
+
+class ImportModuleTest(unittest.TestCase):
+
+    def setUp(self):
+        current_dir = os.path.dirname(os.path.relpath(__file__, '.'))
+        self.non_existing_path = os.path.join(current_dir,
+                                              'importers_test_dir',
+                                              'non_existing.py')
+        self.non_python_path = os.path.join(current_dir,
+                                            'importers_test_dir',
+                                            'test.nonpy')
+
+    def test_non_existing(self):
+        with self.assertRaisesRegex(ImportError, (
```

Comment:
I think the brace at end of line is not necessary^^

(same below)

Reply:
@Makman2 It saves us one line, because otherwise I would have to put `ImportError` on the next line for PEP8 consistency ;)

Reply:
@Makman2 Everything I do has some very special reason ;P

Reply:
... then just put it onto the next line 😅 don't do tricks with codestyles, just ensure it's nice to read :3